docs: update chat endless scrolling logic BLZ-2750 (#3680)
telerik/blazor-docs: components/chat/endless-scrolling.md

## Client-Side Endless Scrolling

> In client-side mode, the component pages the `Data` collection automatically. No additional parameters or event handlers are required. Subscribe to `OnLoadMoreMessages`, if you want to be notified of paging activity — the event fires whenever the rendered message range changes.

Reviewer: ```suggestion
In client-side mode, the component pages the `Data` collection automatically. No additional parameters or event handlers are required. Subscribe to [`OnLoadMoreMessages`](#loading-more-messages), if you want to detect changes in the rendered message range.
```
